src/jump.js

```diff
@@ -19,6 +19,10 @@ export default class Jump {
       ? this.options.duration(this.distance)
       : this.options.duration
 
+    !!this.options.container
+      ?  this.start = this.options.container.getBoundingClientRect().top
+      :  this.start = window.pageYOffset
+
```

Comment:
I adjusted the calculations and removed this part of code, now it should be much smoother. If you have trouble getting it to work please let me know and I'll try to help out.